apps/ios/Shared/Views/Helpers/ComposeLinkView.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  LinkPreview.swift
+//  SimpleX
+//
+//  Created by Ian Davies on 04/04/2022.
+//  Copyright © 2022 SimpleX Chat. All rights reserved.
+//
+
+import SwiftUI
+import LinkPresentation
+
+
+func getLinkPrivew(url: URL, cb: @escaping (LinkPreview?) -> Void) {
+    logger.debug("getLinkMetadata: fetching URL preview")
```

Comment:
Do we still need this line?

Reply:
I don't think there is any harm in keeping debug logs